test/native/setup.js

```diff
@@ -89,3 +89,5 @@ Object.keys( mockNativeModules ).forEach( ( module ) => {
 		jest.doMock( module, () => mockNativeModules[ module ], { virtual: true } );
 	}
 } );
+
+jest.mock( 'Platform', () => ( { OS: '', select: () => '' } ) );
```

Comment:
Why do we need this one for this PR?

Reply:
Ahh no we don't sorry,  I was using `Platform.select` but ended up removing it when I made some changes. I'll update it 👍

Reply:
Removed =)